### DISCUSSION TITLE
Dependency Track BOM

### DISCUSSION THREAD

**Discussion by curnsey** [DependencyTrack/dependency-track#1747] [General]
Maybe someone can point me in the right direction, but is there a BOM file for Dependency Track itself? I find it a bit odd that there isn't one. :)

Thanks.

**Comment by tmehnert** [comment]
Hi @curnsey you can look at the [Releases](https://github.com/DependencyTrack/dependency-track/releases) page. The BOM is collected as an "Asset".

**Reply by curnsey** [reply]
Thank you. Appreciate the response.
